app/views/groups/index.html.erb

```diff
@@ -49,6 +49,11 @@
     }) %>
 <% end %>
 
+<br /><br />
```

Comment:
Oops.. I need to move my new block of code inside the `else` clause above, right?  After this line? `  }) %>` ?

```
<% else %>
  <%= raw t('groups.index.instructions', {
      icon: '<i class="fa fa-plus-circle"></i>'
    }) %>
<% end %>
```